app/controllers/applicants_controller.rb

```diff
@@ -6,6 +6,15 @@ class ApplicantsController < ApplicationController
   def show
     wait_on -> { @applicant.processed? }
   end
+  
+  def index
```

Comment:
So this method already exists - on the `/admin/ApplicantsController`. This application uses a slightly more advanced pattern where the scope is considered in the route. `/applicants` does not have a scope - so it's meant to be 'public' (and it doesn't exist - there probably shouldn't be a 'public' listing of applicants). On the flip side `/admin/applicants` is a completely separate call - showing a list of applicants within the scope of an admin (where it totally should exist). It's likely that as we transition to a case management app, we won't need this distinction, as there will not be distinct scopes (all users will likely be case management workers), but for now this distinction exists.

∴ Let's move this logic to that controller.